### ISSUE TITLE
PDF not opening

### ISSUE THREAD

**Issue by adham-123** [EdgeTranslate/EdgeTranslate#56]
**描述问题**

_请在此详细描述你遇到的问题，例如问题的表现形式，对你造成了那些影响等_

**出现问题的网址**

_如果问题仅在某些特定的网页上出现，请在此给出网页链接_

**复现步骤**

_请在此给出问题的复现步骤_

**问题截图**

_可以的话，最好再给出出现问题时的截图（注意隐藏你的个人信息）_

**使用环境**

_在这里给出你的使用环境，例如：_

_浏览器类型 [e.g. chrome, firefox]_

_浏览器版本 [e.g. 22]_

_插件版本 [e.g. 1.2.0]_

**其他信息**

_如果还有其他相关信息，请补充到这里_

**Comment by nickyc975** [comment]
Which browser are you using? Chrome or Firefox?
PDF related features are unavailable on Firefox. That's Firefox's problem. We can't fix it.

**Comment by adham-123** [comment]
No I'm using Microsoft edge chromium

**Comment by nickyc975** [comment]
What's your Edge Chromium version? I tested on Edge Chromium 80.0.361.66 and did not find any problem.

**Comment by adham-123** [comment]
I found a Bug for you, or dunno what u call it. I’m using the latest version.. But the problem was with the folder that the PDFs was put in. That file was named “ C# “ , so the “ # “ symbol is preventing your extension to open the PDF.When I changed the name to “ C sharp “ just when I removed “ # “ symbol the PDF worked.  Thanks for your work 😊  Sent from Mail for Windows 10 From: Nicky ChenSent: Friday, March 20, 2020 11:31 AMTo: EdgeTranslate/EdgeTranslateCc: adham-123; AuthorSubject: Re: [EdgeTranslate/EdgeTranslate] PDF not opening (#56) What's your Edge Chromium version? I tested on Edge Chromium 80.0.361.66 and did not find any problem.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
